src/cascadia/ShellExtension/ShellExtension.vcxproj

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{f2ed628a-db22-446f-a081-4cc845b51a2b}</ProjectGuid>
+    <ProjectName>ShellExtension</ProjectName>

```

Comment:
Is there something we can use to change the _filename_ of the dll? I'd prefer it to be `Microsoft.Terminal.ShellExtension.dll` rather than `ShellExtension.dll`

Reply:
Much much much prefer.

Reply:
If somebody looks at the loaded module list in explorer and sees "shell extension lol", that'll be funny, but not enlightening.

Reply:
Sure new, that's easy